pytest_tests/testsuites/acl/test_acl.py

```diff
@@ -178,3 +180,180 @@ def test_basic_acl_readonly(self, wallets, client_shell, read_only_container, fi
                 shell=client_shell,
                 cluster=self.cluster,
             )
+
+    @allure.title("Test basic ACL IR and STORAGE rules compliance")
+    def test_basic_acl_ir_storage_rules_compliance(self, wallets, public_container, file_path):
```

Comment:
No type annotations here, at least some can be added

Reply:
> No type annotations here, at least some can be added

Thank you very much! Sorry, I wasn't paying attention.
Fixed.